src/main/java/me/clip/placeholderapi/util/TimeUtil.java

```diff
@@ -20,143 +20,60 @@
-			case SECONDS:
-				return String.valueOf(secondsLeft);
-			}
-			return String.valueOf(seconds);
-		}
-		
-		if (minutes < 1440) {
-			int hours = minutes / 60;
-			int inMins = 60 * hours;
-			int leftOver = minutes - inMins;
-			switch(type) {
-			case DAYS:
-				return "0";
-			case HOURS:
-				return String.valueOf(hours);
-			case MINUTES:
-				return String.valueOf(leftOver);
-			case SECONDS:
-				return String.valueOf(secondsLeft);
-			}
-			return String.valueOf(seconds);
-		}
-		
-		int days = minutes / 1440;
-		int inMins = 1440 * days;
-		int leftOver = minutes - inMins;
-			
-		if (leftOver < 60) {
-			switch (type) {
-			case DAYS:
-				return String.valueOf(days);
-			case HOURS:
-				return String.valueOf(0);
-			case MINUTES:
-				return String.valueOf(leftOver);
-			case SECONDS:
-				return String.valueOf(secondsLeft);
-			}
-			return String.valueOf(seconds);
+import me.clip.placeholderapi.PlaceholderAPIPlugin;
+import me.clip.placeholderapi.configuration.PlaceholderAPIConfig;
+import me.clip.placeholderapi.util.TimeFormat;
 
-		} else {
-			int hours = leftOver / 60;
-			int hoursInMins = 60 * hours;
-			int minsLeft = leftOver - hoursInMins;
-			switch (type) {
-			case DAYS:
-				return String.valueOf(days);
-			case HOURS:
-				return String.valueOf(hours);
-			case MINUTES:
-				return String.valueOf(minsLeft);
-			case SECONDS:
-				return String.valueOf(secondsLeft);
-			}
-			return String.valueOf(seconds);
-		}	
-	}
+public class TimeUtil {
 	
-	public static String getTime(int seconds) {
+	public static final String getRemaining(int seconds, TimeFormat type) {
+		final PlaceholderAPIPlugin plugin = (PlaceholderAPIPlugin) JavaPlugin.getProvidingPlugin(PlaceholderAPIPlugin.class);
+		final PlaceholderAPIConfig config = plugin.getPlaceholderAPIConfig();
+		final String time = config.getTime(type);
+		final int days = seconds / 86400; seconds = seconds - (days * 86400);
+		final int hours = seconds / 3600; seconds = seconds - (hours * 3600);
+		final int minutes = seconds / 60; seconds = seconds - (minutes * 60);
 		
-		if (seconds < 60) {
-			return seconds+"s";
+		switch (type) {
+		case DAYS : return days + time;
+		case HOURS : return hours + time;
+		case MINUTES : return minutes + time;
+		case SECONDS : return seconds + time;
+		default : return null;
 		}
+	}
+
+	public static final String getTime(int seconds) {
+		final PlaceholderAPIPlugin plugin = (PlaceholderAPIPlugin) JavaPlugin.getProvidingPlugin(PlaceholderAPIPlugin.class);
+		final PlaceholderAPIConfig config = plugin.getPlaceholderAPIConfig();
+		final String time = config.getTime(type);
+		final String timeDays = getTime(TimeFormat.DAYS);
+		final String timeHours = getTime(TimeFormat.HOURS);
+		final String timeMinutes = getTime(TimeFormat.MINUTES);
+		final String timeSeconds = getTime(TimeFormat.SECONDS);
+		final StringBuilder timeBuilder = new StringBuilder();
+		final int days = seconds / 86400; seconds = seconds - (days * 86400);
```

Comment:
`seconds -=` instead of `seconds = seconds -`.

Also, putting these statements on the same line as an initialization of a variable greatly decreases readability.